src/Codeception/Module/AMQP.php

```diff
@@ -305,6 +305,55 @@ public function seeMessageInQueueContainsText($queue, $text)
         $this->assertContains($text, $msg->body);
     }
 
+    /**
+     * Count messages in queue.
+     *
+     * @param string $queue
+     *
+     * @return int
+     */
+    public function countMessageInQueue($queue)
+    {
+        list($queue, $messageCount) = $this->getChannel()->queue_declare($queue, true);
+        return $messageCount;
+    }
+
+    /**
+     * Checks if queue contain messages.
+     *
+     * ``` php
+     * <?php
+     * $I->pushToQueue('queue.emails', 'Hello, davert');
+     * $I->seeMessageInQueue('queue.emails');
+     * $I->seeMessageInQueue('queue.emails',1);
+     * ?>
+     * ```
+     *
+     * @param string $queue
+     * @param int $expected
+     */
+    public function seeMessageInQueue($queue, $expected = null)
```

Comment:
I really don't like the name of this method - if I saw it in the test, I would assume that it is checking if specific message is in the queue.
A better name matching Codeception conventions would be `seeQueueIsNotEmpty`
and a more appropriate name for `dontSeeMessageInQueue` is `seeQueueIsEmpty`.

On another hand, it would probably be good to match method names of [Queue module](https://codeception.com/docs/modules/Queue#seeEmptyQueue) - `seeEmptyQueue`, `dontSeeEmptyQueue`, `seeQueueHasCurrentCount`.

Reply:
ok to match Queue module, but if someone have enable `AMQP` and `Queue` there is conflict, or we assume that it will never append ?

Reply:
You are right, using different method names to avoid conflict is better.

Reply:
I've made the changes